SGM/src/sgmstereo/dsimage.h

```diff
@@ -14,6 +14,16 @@
 #include <string.h>
 #include <math.h>
 
+#ifdef __unix__
```

Comment:
Since this seems to be replacing something specific for MSVC compiler, I think the condition should be #ifndef WIN32. This will also cover __ linux __.